black.py

```diff
@@ -504,7 +504,7 @@ def reformat_many(
     report: "Report",
 ) -> None:
     """Reformat multiple files using a ProcessPoolExecutor."""
-    loop = asyncio.get_event_loop()
+    loop = cast(asyncio.BaseEventLoop, asyncio.get_event_loop())
```

Comment:
We should avoid `cast` where possible. Can you change the argument types of schedule_formatting() and shutdown() instead?

Reply:
Good suggestion. Done!